src/mesh/voxel_cloud.rs

```diff
@@ -30,93 +30,158 @@ pub enum FalloffFunction {
     Square(f32),
     /// distance values: 0 (excl) to infinity \
     /// suggested multiplier: 1 \
-    /// output values: 1 to 0 (excl) \
-    /// value on volume: 1
+    /// output values: infinity to 0 \
+    /// value on volume: infinity
     #[allow(dead_code)]
     InverseLinear(f32),
     /// distance values: 0 (excl) to infinity \
     /// suggested multiplier: 1 \
-    /// output values: 1 to 0 (excl) \
-    /// value on volume: 1
+    /// output values: infinity to 0 \
+    /// value on volume: infinity
     InverseSquare(f32),
-    /// http://www.geisswerks.com/ryan/BLOBS/blobs.html \
-    /// distance values: 0 to 1 \
-    /// suggested multiplier: 1 / growth \
-    /// output values: 0 to 1 \
-    /// value on volume: 1
-    #[allow(dead_code)]
-    Perlin(f32),
 }
 
 impl FalloffFunction {
-    pub fn apply(self, distance: f32, is_outside: bool) -> Option<f32> {
-        // FIXME: These will become proper math functions in the very next PR.
+    pub fn apply(self, distance: f32) -> Option<f32> {
         match self {
             FalloffFunction::Linear(mul) => {
                 let d = distance * mul;
-                if d == 0.0 {
-                    Some(0.0)
-                } else {
-                    let value = d;
-                    if is_outside {
-                        Some(value)
-                    } else {
-                        Some(-value)
-                    }
-                }
+                Some(d)
             }
             FalloffFunction::Square(mul) => {
                 let d = distance * mul;
-                if d == 0.0 {
-                    Some(0.0)
-                } else {
-                    let value = d * d;
-                    if is_outside {
-                        Some(value)
-                    } else {
-                        Some(-value)
-                    }
-                }
+                Some(d * d)
```

Comment:
Is that what we were talking about? I understood it this way - only changing the style so that the math functions look cleaner? If we went your way, we can remove the multiplier whatsoever because we already have a function that multiplies all the values.

Reply:
nah, that's not good. we need to multiply the input. 
i've already implemented it, you'll see with the next push